src/groups/bmq/bmqp/bmqp_ctrlmsg_messages.cpp

```diff
@@ -561,49 +578,16 @@ const bdlat_AttributeInfo* CloseQueueResponse::lookupAttributeInfo(int id)
 
 // CREATORS
 
-CloseQueueResponse::CloseQueueResponse()
```

Comment:
What is the reason for this big change in formatting.  `bas_codegen.pl` version/arguments?

Reply:
I am not sure. I cloned the latest version of [https://bbgithub.dev.bloomberg.com/bas/bas-all](url) and used `bas_codegen.pl` from that. When I run `bas_codegen.pl --version` I get `@BLP_BAS_CODEGEN_VERSION@`, which feels like a bug or maybe an error with my dev environment.

The arguments I copied from the prior generated file:
`bas_codegen.pl -m msg --noAggregateConversion --noExternalization --noIdent --package bmqp_ctrlmsg --msgComponent messages bmqp_ctrlmsg.xsd`